### PR TITLE
publish 1.0 docs

### PR DESCRIPTION
**Pull request by pjfanning**
equivalent of what we have in other repos

### CODE REVIEW

File: .github/workflows/publish-1.0-docs.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        uses: actions/checkout@v3
+        with:
+          # we don't know what commit the last tag was it's safer to get entire repo so previousStableVersion resolves
+          fetch-depth: 0
+
+      - name: Setup Java 8
+        uses: actions/setup-java@v3
+        with:
+          distribution: temurin
+          java-version: 8
```
Comment (He-Pin) [inline .github/workflows/publish-1.0-docs.yml:28]: Not 11?

Comment (pjfanning) [reply .github/workflows/publish-1.0-docs.yml:28]: this is what we use in the nightly build - I don't want to use a different java version in this build

Comment (mdedetrich) [reply .github/workflows/publish-1.0-docs.yml:28]: iirc all of our projects aside from pekko core (due to technical reasons) are built with JDK 8

**Review by mdedetrich — APPROVED** [review]
lgtm

**Pull request merged**